scripts/test-server.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
What's this for? Running the tests in the browser? If so, thank you very much!
But why do we need the xvfb?